src/test/java/uk/gov/justice/hmpps/prison/service/CaseNoteServiceImplIntTest.java

```diff
@@ -26,13 +28,36 @@
     @MockBean
     private UserService userService;
 
+    @SuppressWarnings("unused")
+    @MockBean
+    private BookingService bookingService;
+
     @SuppressWarnings("unused")
     @MockBean
     private AuthenticationFacade authenticationFacade;
 
     @Autowired
     private CaseNoteService caseNoteService;
 
+    @Test
+    @WithMockUser(username = "ITAG_USER", roles = {"CASE_NOTE_EVENTS"})
+    public void createCaseNote_maximumTextSizeExceededDueToUtf8() {
+
+        String stringWith10Chars = "ABCDE12345";
```

Comment:
Use final var to declare types unless it's not possible or hurts readability.

Reply:
Ah yes